src/settings/RequestCancellationReasons.js

```diff
@@ -30,7 +32,14 @@ class RequestCancellationReasons extends React.Component {
         visibleFields={['name', 'description', 'publicDescription']}
         hiddenFields={['lastUpdated', 'numberOfObjects']}
         columnMapping={{
-          name: <FormattedMessage id="ui-circulation.settings.cancelReasons.labelShort" />,
+          name: (
+            <Label
+              className={css.header}
```

Comment:
I don't think this is necessary any longer now that 
[STCOM-565](https://issues.folio.org/browse/STCOM-565) / folio-org/stripes-components/pull/1025 is in place. Can we remove it?

Reply:
Done